hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -814,6 +814,11 @@ public void close() throws IOException {
     IOUtils.cleanupWithLogger(LOG, xceiverClientManager);
   }
 
+  @VisibleForTesting
+  public XceiverClientManager getXceiverClientManager() {
```

Comment:
NIT: should we expose the XceiverClientMetris here instead of XceiverClientManager in RpcClient? It can be consumed directly by the callers (many of them are unit tests)?